ui/Screen/Settings.svelte

```diff
@@ -19,176 +25,48 @@
 </script>
 
 <style>
-  .container {
+  main {
     max-width: var(--content-max-width);
     margin: 64px auto;
     min-width: var(--content-min-width);
     padding: 0 var(--content-padding);
   }
-  section header {
+
+  main :global(section header) {
```

Comment:
It seems odd that the styling is now detached from the actual component it styles. To me it would make more sense to extract `<Section>` and `<Header>` as separate components into `ui/Screen/Settings/` and use them in all the sub-screens as wrappers.